flang/test/Driver/emit-llvm-all-drivers.f90

```diff
@@ -0,0 +1,22 @@
+! RUN: %flang_fc1 -emit-llvm %s -o %t.flang | FileCheck %s
+! RUN: %flang_fc1 -emit-mlir %s -o %t.flang_tco | tco | FileCheck %s
+! RUN: bbc %s -o - | tco -o %t.bbc_tco | FileCheck %s
+
+! This is added as a regression test for a failure reported in https://github.com/flang-compiler/f18-llvm-project/issues/1280. It
+! makes sure that this example can be lowered to LLVM IR by all major drivers in Flang. Ultimately, we should be veryfying that the
+! generated code is identical in all the tested cases. Currently, there are some discrepancy in the the generated debug info.
+! TODO: Make sure that the generated code is identical in all cases
```

Comment:
Looks like more than 80.

Reply:
It is. Don’t mind updating, though the 80 char limit is rarely applied to test files.

Reply:
Up to you.

Reply:
Updated the comment.

Also, fixed the test :) (sorry, was messing around and forgot to restore the original version before sending to GitHub - should be all good now!)